addons/wowl/static/src/components/navbar/navbar.ts

```diff
@@ -9,13 +9,21 @@ export class NavBar extends Component<{}, OdooEnv> {
   state = useState({ showDropdownMenu: false });
 
   systrayItems = this.env.registries.systray.getAll();
+  constructor(...args: any[]) {
+    super(...args);
+    this.env.bus.on("menus:app-changed", this, () => this.render());
```

Comment:
FYI, I changed `action_manager:update` to `ACTION_MANAGER:UPDATE` as it seems that there was some sort of convention to write (main bus) events in uppercase